src/account-info/AccountDetailsScreen.js

```diff
@@ -71,6 +95,19 @@ class AccountDetailsScreen extends PureComponent<Props> {
 }
 
 export default connect<SelectorProps, _, _>((state, props) => ({
-  user: getAccountDetailsUserForEmail(state, props.navigation.state.params.email),
-  isActive: getUserIsActive(state, props.navigation.state.params.email),
+  user: (() => {
+    const { userId } = props.navigation.state.params;
+    return userId === undefined ? userId : getAllUsersById(state).get(userId);
+  })(),
+  isActive: (() => {
+    const { userId } = props.navigation.state.params;
+    if (userId === undefined) {
+      return false;
+    }
+    const user = getAllUsersById(state).get(userId);
+    if (!user) {
+      return false;
+    }
+    return getUserIsActive(state, user.email);
+  })(),
```

Comment:
For future reference – `isActive` effectively needs the value of `user`, so you might as well make that explicit (and also avoid recomputing it). For example:

```js
... connect((state, props) => {
  const { userId } = props.navigation.state.params;
  const user: number | void = ...
  const isActive: boolean = ...
  return { user, isActive };
}))(...
```

(I say "for future reference" since I'm aware most of this goes away in the next commit. :upside_down_face: No need to take action here.)